easy_my_coop/models/operation_request.py

```diff
@@ -200,6 +200,14 @@ def validate(self):
                 raise ValidationError(_("This share type could not be"
                                         " transfered to " +
                                         self.partner_id_to.name))
+            if not self.receiver_not_member and self.partner_id_to.is_company \
```

Comment:
as said for https://github.com/coopiteasy/vertical-cooperative/pull/78 not sure that is you should test self.receiver_not_member